homepage/homepage/content/docs/project-setup/ssr.mdx

```diff
@@ -0,0 +1,307 @@
+import { Alert } from "@garden-co/design-system/src/components/atoms/Alert";
+import { CodeGroup, ContentByFramework, FileName, TabbedCodeGroup, TabbedCodeGroupItem } from "@/components/forMdx";
+import { SvelteLogo } from "@/components/icons/SvelteLogo";
+import { ReactLogo } from "@/components/icons/ReactLogo";
+import RunDevServer from '@/components/docs/snippets/RunDevServer.mdx'
+import StuckCTA from '@/components/docs/snippets/StuckCTA.mdx'
+
+export const metadata = {
+  description: "Add server-side rendering to your Jazz app."
+};
+
+# Add Server-Side Rendering to your App
+
+This guide will take your simple client-side app to the next level by showing you how to create a server-rendered page to publish your data to the world.
+
+<Alert variant="info" className="mt-4 flex gap-2 items-center">
+  If you haven't gone through the [front-end Quickstart](/docs/getting-started/quickstart), you might find this guide a bit confusing. If you're looking for a quick reference, you might find [this page](/docs/project-setup#ssr-integration) more helpful!
+</Alert>
+
+## Creating an agent
+For Jazz to access data on the server, we need to create an SSR agent, which is effectively a read-only user which can access public data stored in Jazz.
+
+We can create this user using the `createSSRJazzAgent` function. In this example, we'll create a new file and export the agent, which allows us to import and use the same agent in multiple pages.
+
+<ContentByFramework framework="react">
+<FileName>app/jazzSSR.ts</FileName>
+</ContentByFramework>
+<ContentByFramework framework="svelte">
+<FileName>src/lib/jazzSSR.ts</FileName>
+</ContentByFramework>
+<CodeGroup className="mt-4 [&_span]:[tab-size:2]" preferWrap>
+```tsx
+import { createSSRJazzAgent } from "jazz-tools/ssr";
+
+export const jazzSSR = createSSRJazzAgent({
+  peer: "wss://cloud.jazz.tools/",
+});
+```
+</CodeGroup>
+
+## Telling Jazz to use the SSR agent
+
+Normally, Jazz expects a logged in user (or an anonymous user) to be accessing data. We can use the `enableSSR` setting to tell Jazz that this may not be the case, and the data on the page may be being accessed by an agent.
```

Comment:
Is there a page in the docs where we explain the difference between accounts & agents in more detail? If we do, maybe we could link it here